.github/workflows/pull-request-checks.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Pull Request Checks'
+
+on:
+  workflow_dispatch:
+
+jobs:
+  find-pull-request:
+    name: Find pull request target
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:latest
+    defaults:
+      run:
+        shell: bash
+    outputs:
+      pull-request-target: ${{ steps.pull-request-target.outputs.results }}
+    steps:
+      - name: Setup container
+        run: apt update && apt install -y git
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - name: Find pull request target
+        id: pull-request-target
+        uses: ./.github/actions/find-pull-request
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Can/should this just inherit the token from the github secrets instead of passing as an input?  Does it even matter?

Reply:
Composite actions do not inherit secrets. They have to be passed in as inputs.